nixos/modules/image/repart.nix

```diff
@@ -211,6 +211,15 @@ in
       '';
     };
 
+    finalPartitions = lib.mkOption {
```

Comment:
What's the benefit of making this an option over exposing it via `passthru`?

Reply:
I don't think there is any. This way seems more intuitive to me, specifying an option as a single source of truth and referencing that where needed. The difference from a user perspective would be `config.image.repart.finalPartitions` vs. `config.image.repart.image.passthru.finalPartitions`. I don't see any strong argument for either approach %)